concepticondata/conceptlists/Castro-2015-608.tsv

```diff
@@ -266,7 +266,7 @@ Castro-2015-608-264	264	father	1217	FATHER	父亲	father
 Castro-2015-608-265	265	mother	1216	MOTHER	母亲	mother
 Castro-2015-608-266	266	maternal uncle (elder)	3249	MATERNAL UNCLE (MOTHER'S OLDER BROTHER)	大舅（妈之兄）	uncle
 Castro-2015-608-267	267	maternal uncle (younger)	3250	MATERNAL UNCLE (MOTHER'S YOUNGER BROTHER)	小舅（妈之弟）	uncle
-Castro-2015-608-268	268	maternal aunt			舅母（妈之姐）	mother's older sister or older brother's wife
+Castro-2015-608-268	268	maternal aunt	2176	MATERNAL AUNT	舅母（妈之姐）	mother's older sister or older brother's wife
```

Comment:
This one is a bit tricky due to the very intricate system of how people are addressed in Chinese. The gloss in other lists is mapped to MATERNAL AUNT (MOTHER'S SISTER), MATERNAL AUNT (WIFE OF MOTHER'S BROTHER), or PATERNAL AUNT. @mathildavz Could you open an issue where we discuss how to map the gloss consistently?

Reply:
Of course! Should I unmap it for now? @AnnikaTjuka

Reply:
Yes, I'd unmap this for now.

Reply:
I'll take care of it immediately.